client/environment_import.go

```diff
@@ -2,44 +2,44 @@ package client
 
 type EnvironmentImport struct {
```

Comment:
omit empty isn't needed here. This is just a response (doesn't matter to in golang).

Reply:
[9e15d2d](https://github.com/env0/terraform-provider-env0/pull/855/commits/9e15d2dadffef2494868428e5dbde521b3e2431d)